src/frontend/src/pages/UpdatePage.js

```diff
@@ -0,0 +1,144 @@
+import {Blog} from "../components/Blog";
+import {LoadingDialog} from "../components/LoadingDialog";
+
+export const BlogPage = ()=>{
+
+    
+    const {id} = useParams()
+    const [blog,setBlog] = useState(null)
+    const [loading,setLoading] = useState(false)
+    let navigate = useNavigate()
+
+    const [title,setTitle] = useState("")
+    const [content,setContent] = useState("")
+    const [value,setValue] = useState(0)
+    const [width,setWidth] = useState(window.innerWidth)
+
+    useEffect(()=>{
+        const handleWidth = ()=>{
+            setWidth(window.innerWidth)
+        }
+        window.addEventListener("resize",handleWidth)
+        return ()=>{
+            window.removeEventListener("resize",handleWidth)
+        }
+    })
+    useEffect(()=>{
+        setLoading(true)
+        axios.get(`${process.env.REACT_APP_API_BASE_URL}/blog/${id}`,{
+            method : "GET"
+        })
+            .then(response=> setBlog(response.data))
+            .catch(error=> console.log(error))
+            .finally(()=> setLoading(false))
+    },[])
+
+    const changeTitle = (event)=>{
+        setTitle(event.target.value)
+    }
+
+    const changeContent = (event)=>{
+        setContent(event.target.value)
+    }
+
+    const changeValue = (event,newValue)=>{
+        setValue(newValue)
+    }
+
+    const handleUpdate = ()=>{
+        if(title==="" || content===""){
+                toast.error("One or more fields are empty", {
+                position: toast.POSITION.TOP_CENTER
+            });
+            return
+        }
+        axios({
+            url : `${process.env.REACT_APP_API_BASE_URL}/blog/${id}`,
+            method : "PUT",
+            headers : {
+                "Content-Type" : "application/json",
+                "Authorization" : `Bearer ${localStorage.getItem("jwtToken")}`
+            },
+            data : {
+                "title" : title,
+                "content" : content
+            }
+        })
+            .then(
+                navigate(`/blog/${id}`)
+            )
+            .catch(error=>{
+                console.log(error)
+                toast.error(error.response.data, {
+                    position: toast.POSITION.TOP_CENTER
+                });
+            })
+    }
+
+    return (
+        <div>
+            <div className="flex flex-row items-center justify-center md:flex-col md:mx-[10px]">
+                <div className="text-[45px] font-bold my-[20px] md:text-center sm:text-[30px]">
+                   Update Your Blog
+                </div>
+                <Lottie
+                    play
+                    loop
+                    animationData={writerAnimation}
+                    className="h-[80px] md:h-[150px] sm:h-[100px]"
+                    />
+            </div>
+            <div className="mx-[20px] my-[10px] sm:mx-[10px]">
+                <TextField
+                    variant="outlined"
+                    value={blog.title}
```

Comment:
the value here should be `title` , you can set the title to be the blog title when making the get req above, this way initially the title displayed would be the original title, the same can be done for content